snippets/rust.snippets

```diff
@@ -53,39 +74,33 @@ snippet crate
 	#![comment = "${4:Comment.}"]
 	// Specify the output type
 	#![crate_type = "${5:lib}"]
-snippet allow
-	#[allow(${1:unused_variable})]
-snippet feat
-	#![feature(${1:macro_rules})]
-snippet der "#[deriving(..)]" b
-	#[deriving(${1:Show})]
```

Comment:
`deriving` [has become `derive`][1] and the `Show` trait's functionality [moved to `Debug`][2].

[1]: https://github.com/rust-lang/rust/issues/20362
[2]: http://stackoverflow.com/questions/27874987/why-does-deriveshow-not-work-anymore